bazel/foreign_cc/curl-revert-cmake-minreqver.patch

```diff
@@ -5,13 +5,13 @@ diff --git a/CMakeLists.txt b/CMakeLists.txt
 index b13616fc7..8b6d77542 100644
 --- a/CMakeLists.txt
 +++ b/CMakeLists.txt
-@@ -38,8 +38,7 @@
+@@ -37,8 +37,7 @@
  # To check:
  # (From Daniel Stenberg) The cmake build selected to run gcc with -fPIC on my box while the plain configure script did not.
  # (From Daniel Stenberg) The gcc command line use neither -g nor any -O options. As a developer, I also treasure our configure scripts's --enable-debug option that sets a long range of "picky" compiler options.
--cmake_minimum_required(VERSION 3.0...3.16 FATAL_ERROR)
+-cmake_minimum_required(VERSION 3.2...3.16 FATAL_ERROR)
 -
-+cmake_minimum_required(VERSION 3.0 FATAL_ERROR)
++cmake_minimum_required(VERSION 3.2 FATAL_ERROR)
```

Comment:
Is this a Windows specific patch? If so wondering if a comment should be added to https://github.com/envoyproxy/envoy/blob/master/bazel/repositories.bzl#L677-L678

Reply:
It would be good to understand why this is needed.